docs/en/Storage.md

```diff
@@ -45,10 +45,13 @@ We have four available types of bucket storage.
 
 **Self-Serve:**
 
-- [Minimal](https://minimal-tenant1-minio.covid.cloud.statcan.ca)
-- [Premium](https://premium-tenant1-minio.covid.cloud.statcan.ca)
-- [Pachyderm](https://pachyderm-tenant1-minio.covid.cloud.statcan.ca)
+| storage type                                                        | description                                                                 |
+|:--------------------------------------------------------------------|:----------------------------------------------------------------------------|
+| [Minimal](https://minimal-tenant1-minio.covid.cloud.statcan.ca)     | By default, use this one. It is HDD backed storage.                         |
+| [Premium](https://premium-tenant1-minio.covid.cloud.statcan.ca)     | Use this if you need very high read/write speeds, like for neural networks. |
```

Comment:
More of a side question: How do NN's benefit from premium?  Don't the models sit in memory anyway?  Maybe this is more for any ML model that is training on data that can't be kept in memory (like images that are being picked up and put back down repeatedly)?

Reply:
Yeah images are exactly the use-case I was thinking of

Reply:
ah ok that makes sense.  I suggest saying that directly then.  Something like "this is if you need high read/write speeds.  For example, training a model using a large dataset that does not fit in RAM, such as a large set of images".